colcon_core/shell/template/prefix_util.py

```diff
@@ -38,6 +38,9 @@ def get_packages(prefix_path, merged_install):
     # must match colcon_core.location.get_relative_package_index_path()
     subdirectory = 'share/colcon-core/packages'
     if merged_install:
+        # return if workspace is empty
+        if not (prefix_path / subdirectory).is_file():
```

Comment:
Isn't this condition always going to be `True` since the tested path is supposed to be a directory? Shouldn't it be checking `is_dir()` instead?

Reply:
yes, already fixed